.github/workflows/update-versions.yml

```diff
@@ -0,0 +1,58 @@
+name: Update versions
+
+on:
+    workflow_run:
+        workflows: ['Create release']
+        types:
+            - completed
+
+jobs:
+    check-version-type:
+        runs-on: ubuntu-latest
+        outputs:
+            pre-release: ${{ steps.check-pre-release.outputs.pre-release }}
+        steps:
+            - name: Fetch from origin repo
+              uses: actions/checkout@v3
+              with:
+                  ref: main # always fetch from main branch
+                  fetch-depth: 0
+                  token: ${{ secrets.GHACTIONS }}
+
+            - name: Setup Node.js and Cache
+              uses: ./.github/actions/nodejs
+
+            - name: Check if pre release
+              id: check-pre-release
+              run: |
+                  PRE_RELEASE=$(echo "${{ github.event.workflow_run.head_commit.message }}" | grep -e "-rc." -c) || :
+                  if [ $PRE_RELEASE -eq 0 ]; then
+                    echo "Not a pre-release, continuing..."
+                  else
+                    echo "Pre-release detected, skipping..."
+                  fi
+                  echo "::set-output name=pre-release::$PRE_RELEASE"
```

Comment:
This can be small JS action, which will just check semver.prerelease

Reply:
or you can use actions/github-script@v6

Reply:
i think once we need it somewhere else we can extract this logic to an action.